examples/with-msw/pages/_app.js

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react'
+
+const IS_SERVER = typeof window === 'undefined'
+
+export default function App({ Component, pageProps }) {
+  if (process.env.NODE_ENV === 'development') {
+    const [isReady, setReady] = useState(IS_SERVER)
+
+    useEffect(() => {
+      if (!IS_SERVER) {
```

Comment:
`useEffect` is only called in the client, we can drop this conditional

Reply:
Thanks for spotting this! Will update.

Reply:
Resolved in bf599fd54.